packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -232,6 +233,9 @@ export async function launch(
       blockHeight,
       blockTime,
     });
+    if (setActivityhash) {
```

Comment:
Out of curiosity, why is this not a part of the above `bootstrapBlock` function as well?

Reply:
Hm, good question, I hadn't thought about it. I think that'd be fine, it would capture/signal any divergence during the initial `controller.run()` that sets up the javascript world. Without it, we'd signal that divergence during the second block instead of during the first. The activityhash is updated for every `commitCrank` (including the one that `initializeKernel()` does, making its starting value effectively a hash of the config object), so even if we don't read it at that point, it's still sensitive to the initial state.

I'll add that.